app/vector_icons/BUILD.gn

```diff
@@ -55,8 +55,8 @@ aggregate_vector_icons("brave_vector_icons") {
       "sidebar_reading_list.icon",
       "sidebar_settings.icon",
       "sidebar_trash.icon",
-      "vertical_tab_collapse_button.icon",
-      "vertical_tab_expand_button.icon",
+      "vertical_tab_strip_ntb.icon",
```

Comment:
Out of curiousity, are either of these icons in Leo? You could swap to using the Leo icons, and that way they'll get updated when the designers change them :)

https://leo.bravesoftware.com/?path=/story/icon--all-icons

(you can add them to the `leo_icons` array in `brave/app/vector_icons/BUILD.gn` if Leo has the ones you want)

Reply:
Great! Done